src/main/cpp/src/get_json_object.cu

```diff
@@ -104,25 +104,29 @@ class json_generator {
     }
   }
 
-  CUDF_HOST_DEVICE json_generator finish_child_generator(json_generator const& child_generator)
-  {
-    // logically delete child generator
-    output_len += child_generator.get_output_len();
-  }
+  CUDF_HOST_DEVICE void write_start_array() {}
 
-  CUDF_HOST_DEVICE void write_start_array()
-  {
-    // TODO
-  }
+  CUDF_HOST_DEVICE void write_end_array() {}
 
-  CUDF_HOST_DEVICE void write_end_array()
+  CUDF_HOST_DEVICE void copy_current_unescaped_text(json_parser<max_json_nesting_depth>& parser)
   {
-    // TODO
+    if (nullptr != output) {
+      int len = parser.write_unescaped_text(output + output_len);
+      output_len += len;
+    } else {
+      int len = parser.compute_unescaped_text(output + output_len);
```

Comment:
Why exactly does the `compute_unescaped_text` take an output at all?

Reply:
Done.

Now: 
compute_unescaped_len()